boostlook.css

```diff
@@ -24,36 +24,96 @@
  *   `.boostlook .nav-container` selector targets elements within the `.nav-container` class,
  *   both of which are specific to the Antora HTML structure.
  *
- * - For AsciiDoc templates:
+ * - For AsciiDoctor templates:
  *   The `.boostlook:not(:has(.doc))` selector is used to target elements that do not have 
- *   a `.doc` child, applying styles specific to AsciiDoc content.
+ *   a `.doc` child, applying styles specific to AsciiDoctor content.
 
  *   Since the `.doc` class is not present in the structure above, the styles defined for 
  *   `.boostlook:not(:has(.doc))` will apply.
  */
 
+/*----------------- Root Variables - Start -----------------*/
+
 :root {
-  --bl-pasteboard-color: rgb(209, 213, 219);
-  --bl-pre-background: rgb(248, 248, 248);
-  --bl-table-border-color: rgb(232, 232, 232);
-  --bl-table-head-background: rgb(248, 248, 248);
-  --bl-background: rgb(255, 255, 255);
-  --bl-card-background-color: rgb(255, 255, 255);
-  --bl-cyan-color: rgb(0, 90, 156);
+  --light-bl-pasteboard-color: rgb(209, 213, 219);
+  --light-bl-pre-background: rgb(248, 248, 248);
+  --light-bl-table-border-color: rgb(232, 232, 232);
+  --light-bl-table-head-background: rgb(248, 248, 248);
+  --light-bl-background: rgb(255, 255, 255);
+  --light-bl-card-background-color: rgb(255, 255, 255);
+  --light-bl-cyan-color: rgb(0, 90, 156);
+  --light-bl-quote-background: rgb(250, 250, 250);
+  --light-bl-quote-background: rgb(255, 255, 255);
+  --light-bl-code-background: rgb(255, 255, 255);
```

Comment:
why are there 2 of --light-bl-quote-background?

Reply:
Good catch! Per our discussion, I'll fix this in the next one